client/src/components/Home.js

```diff
@@ -65,60 +64,93 @@ const Home = ({ user, logout }) => {
   const postMessage = (body) => {
     try {
       const data = saveMessage(body);
-
-      if (!body.conversationId) {
-        addNewConvo(body.recipientId, data.message);
-      } else {
-        addMessageToConversation(data);
-      }
-
-      sendMessage(data, body);
+      data.then(res => {
+        if (!body.conversationId) {
+          addNewConvo(body.recipientId, res.message);
+        } else {
+          addMessageToConversation(res, body.recipientId);
+        }
+        sendMessage(res, body);
+      });
```

Comment:
This works, however if you look at the function, because it has a try/catch block, it would be better to use the `async/await` syntax instead of the `then/catch` syntax. It would also be more consistent with the rest of the code base.